server/apps/immich/src/modules/immich-jwt/immich-jwt.service.ts

```diff
@@ -13,7 +15,23 @@ export type JwtValidationResult = {
 export class ImmichJwtService {
   constructor(private jwtService: JwtService) {}
 
-  public async generateToken(payload: JwtPayloadDto) {
+  public getCookies(loginResponse: LoginResponseDto) {
+    const maxAge = 7 * 24 * 3600; // 7 days
+
+    const accessTokenCookie = `immich_access_token=${loginResponse.accessToken}; HttpOnly; Path=/; Max-Age=${maxAge}`;
```

Comment:
Maybe use Secure for the token as well? Why do you put it in a Cookie while the old version does not? Local/session storage makes more sense here, or within the mobile app (cookies in apps make no sense).